SpringProject/src/main/resources/db/migration/V2__create_schema.sql

```diff
@@ -5,5 +5,6 @@ CREATE TABLE users (
     last_name VARCHAR(255),
     email VARCHAR(255),
     password CHAR(60),
-    occupation VARCHAR(255)
+    occupation VARCHAR(255),
```

Comment:
As you are wiping your database every time you'll get away with this - but this is not how flyway works - you're meant to have another migration script that updates your table to add these new columns (meant for auditing purposes I think) - be careful